content-src/components/ContextMenu/ContextMenu.scss

```diff
@@ -33,8 +34,33 @@
         &:hover {
           background: $star-blue;
           color: $white;
+
+          a {
+            color: $dark-grey;
+          }
+
+          a:hover {
+            color: $white;
+          }
         }
       }
     }
   }
 }
+
+.submenu {
+  position: relative;
+
+  .context-menu {
+    display: none;
+    left: auto;
+    right: 95%;
```

Comment:
Right now the share sub menu always opens to the left. I would love it to open to the right when we know there is enough space. But I have no idea how to do that. Help?